packages/trip-form/src/TripOptions/styled.ts

```diff
@@ -0,0 +1,259 @@
+import { CheckCircle } from "@styled-icons/boxicons-solid/CheckCircle";
+import { PlusCircle } from "@styled-icons/boxicons-regular/PlusCircle";
+import ScrollContainer from "react-indiana-drag-scroll";
+import styled, { css } from "styled-components";
```

Comment:
See my comment regarding sorting imports while ignoring @.

Reply:
Corrected in 775500dafb258ffc2fc5d431b8619ebe3f21b54f. I think this should be resolved using lint rules. Have added a ticket to modify our config